app/controllers/application_controller.rb

```diff
@@ -3,6 +3,11 @@ class ApplicationController < ActionController::Base
 
   include RealIpLogger
 
+  # Error modal contact message
+  ZENDESK_CONTACT = "If this error persists and you require assistance, please contact support at " \
+                    "<a href='mailto:scp-support@broadinstitute.zendesk.com' id='zendesk-alert-email'>" \
```

Comment:
this should probably be a class, not an id, since we could want to have multiple alerts on a given page

Reply:
also, I could see giving this element a data-analytics-name of "SUPPORT_EMAIL" just so it's easier to find in mixpanel, but I defer to Eric on naming questions

Reply:
> a data-analytics-name of "SUPPORT_EMAIL"

Let's use kebab-case, per [convention](https://github.com/broadinstitute/single_cell_portal_core/search?q=data-analytics-name%3D%22).

Reply:
390a54e